doc/articles/getting-started/GettingStartedWithTests.md

```diff
@@ -0,0 +1,61 @@
+---
+uid: Uno.Authoring.Tests
+---
+# How-To: Creating an application with tests using Uno.Extensions
```

Comment:
Is the mention of "Uno.Extensions" here needed?

cc @jeromelaban @agneszitte @dansiegel

Reply:
No, it's a generic feature. `How-To: Creating an application with tests` will do.